# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxPlatformFontList.h
# Commit: 62f8ce89dce8
# Full Hash: 62f8ce89dce8f1162ff2f57debdd9a756a11a2f9
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2020-04-03 01:02:14
# Regressor Bug: 1619349
# File Overlap Count: 1
# Description:
#   Bug 1619349 - patch 1 - Record statistics about font-matching behavior in the presContext. r=jwatt
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D67787
# ==============================================================================

diff -r bcbee7191130 -r 62f8ce89dce8 gfx/thebes/gfxPlatformFontList.h
--- a/gfx/thebes/gfxPlatformFontList.h	Thu Apr 02 01:38:21 2020 +0000
+++ b/gfx/thebes/gfxPlatformFontList.h	Thu Apr 02 01:38:18 2020 +0000
@@ -208,7 +208,8 @@
 
   gfxFontEntry* SystemFindFontForChar(uint32_t aCh, uint32_t aNextCh,
                                       Script aRunScript,
-                                      const gfxFontStyle* aStyle);
+                                      const gfxFontStyle* aStyle,
+                                      FontVisibility* aVisibility);
 
   // Flags to control optional behaviors in FindAndAddFamilies. The sense
   // of the bit flags have been chosen such that the default parameter of